core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.ts

```diff
@@ -366,8 +366,6 @@ export class StoryEditorNavbarComponent implements OnInit {
           this.story, this.storyNode.getId(), oldStatus);
       }
     );
-    this.storyEditorStateService.loadStory(this.story.getId());
```

Comment:
Just to confirm, is it guaranteed that dropping these lines will still preserve any changes that were made in the UI?

Reply:
Yes, there is no need to reload the changed things again.